src/templates/faq-page.js

```diff
@@ -28,10 +30,21 @@ export const FaqPageTemplate = ({
       <PageSection id={sections.faq.QUESTIONS}>
         <Row className="questions-section">
           <Col xs={12} sm={12}>
-            {questions.map((item, i) => (
-              <div key={`question-${i}`} className="faq-question">
-                <p>{item.question}</p>
-                <Image imageData={plusIcon} />
+            {questions.map((item, index) => (
```

Comment:
@maymax777, There is a `CollapsibleList` component that already defined in `components`. We can use it for this questions list.